Packs/Flashpoint/ReleaseNotes/2_0_0.md

```diff
@@ -0,0 +1,136 @@
+
+#### Mappers
+
+##### New: Ignite Alert - Incoming Mapper
+
+- New:Added the new incoming mapper for the "Ignite Alert" incident type (Available from Cortex XSOAR 6.10.0).
+
+
+#### Incident Types
+
+##### New: Ignite Alert
+
+- New: **Ignite Alert**
+
+
+#### Layouts
+
+##### New: Ignite Alert
+
+- New: Added the new layout for the "Ignite Alert" incident type (Available from Cortex XSOAR 6.10.0).
+
+
+#### Incident Fields
+
+##### Flashpoint Alert ID
+
+- Added the "Ignite Alert" incident type.
+
+##### Flashpoint Alert Text
+
+- Added the "Ignite Alert" incident type.
+
+##### New: Flashpoint Basetypes
+
+##### New: Flashpoint Container
+
+##### Flashpoint Created Date
+
+- Added the "Ignite Alert" incident type.
+
+##### New: Flashpoint Data Type
+
+##### New: Flashpoint Entity ID
+
+##### New: Flashpoint Entity Name
+
+##### New: Flashpoint Entity Type
+
+##### New: Flashpoint Generated Date
+
+##### New: Flashpoint Image Safe Search
+
+##### New: Flashpoint Media Storage URI
+
+##### New: Flashpoint Media Type
+
+##### New: Flashpoint Mime Type
+
+##### New: Flashpoint Parent Basetypes
+
+##### New: Flashpoint Parent Data Type
+
+##### New: Flashpoint Phash
+
+##### New: Flashpoint Ports
+
+##### New: Flashpoint Read Status
+
+##### New: Flashpoint Reason ID
+
+##### New: Flashpoint Reason Name
+
+##### New: Flashpoint Reason Origin
+
+##### New: Flashpoint Reason Sources
+
+##### New: Flashpoint Reason Text
+
+##### New: Flashpoint Reason Type
+
+##### New: Flashpoint Resource URL
+
+##### New: Flashpoint Services
+
+##### New: Flashpoint Shodan Host
+
+##### New: Flashpoint Site Actor Name
+
+##### New: Flashpoint Site Name
+
+##### New: Flashpoint Site Native ID
+
+##### Flashpoint Source
+
+- Added the "Ignite Alert" incident type.
+
+##### Flashpoint Source File
+
+- Added the "Ignite Alert" incident type.
+
+##### New: Flashpoint Source ID
+
+##### New: Flashpoint Source Name
+
+##### Flashpoint Source Owner
+
+- Added the "Ignite Alert" incident type.
+
+##### Flashpoint Source Repo
+
+- Added the "Ignite Alert" incident type.
+
+##### New: Flashpoint Source Snippet
+
+##### New: Flashpoint Source Sort Date
+
+##### New: Flashpoint Status
+
+
+#### Integrations
+
+##### Flashpoint (Deprecated)
+
+- Deprecated. Use Flashpoint Ignite instead.
+
+##### New: Flashpoint Ignite
+
+- New: Use the Ignite integration to reduce business risk. Ignite allows users to ingest alerts and compromised credentials as incident alerts and executes commands such as search intelligence report, ip, url, get events, and more.
+
+
+#### Playbooks
+
+##### Compromised Credentials Match - Flashpoint
+
+- Updated the condition for checking the enabled integration of Ignite instead of the deprecated Flashpoint integration.
```

Comment:
```suggestion
Deprecated. Use Flashpoint Ignite instead.

##### New: Flashpoint Ignite

New: Use the Ignite integration to reduce business risk. Ignite allows users to ingest alerts and compromised credentials as incident alerts and execute commands such as search intelligence report, ip, url, get events, and more.


#### Playbooks

##### Compromised Credentials Match - Flashpoint

Updated the condition for checking the enabled integration of Ignite instead of the deprecated Flashpoint integration.
```